.github/workflows/cibuild.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Thank you - that made me thinking. I've added back 3.7. It's temporarily to make it less painful for people with older system to access the new VOSpace services. It worked so I guess it's still OK but it will be removed in the next version.
In addition, I've added support for 3.12. Luckily there wasn't much required for that so I've just squeezed it in.